packages/boilerplate/composables/src/useCart/index.ts

```diff
@@ -5,9 +5,9 @@ import {
   useCartFactory,
   UseCartFactoryParams
 } from '@vue-storefront/core';
-import { Cart, CartItem, Coupon, Product } from '../types';
```

Comment:
Is the `Coupon` object used anywhere else? If not, it can be removed.

Reply:
Its removed already :)